resources/js/components/Kiosk.vue

```diff
@@ -431,6 +430,7 @@ export default {
             });
 
             echo.connector.pusher.connection.bind('state_change', function(states) {
+                Sentry.captureMessage('WebSocket Connection Error', 'warning')
```

Comment:
This sends a websocket connection error even on state changes that aren't issues.  It would be better to stick this in the conditional below that catches the websocket being unavailable.  There are also other connection statuses we can theoretically catch, but unavailable means the websocket is down or unreachable.